exams/2020N/2020N-ronda1.md

```diff
@@ -50,11 +52,12 @@ Script de criação de base de dados e preenchimento com dados, perguntava o que
 
 ## Pergunta 11
 
-Sobre vistas
+Sobre vistas atualizáveis (ver bibliografia)
```

Comment:
Sobre que tipo de vistas atualizáveis? Qual é o enunciado da pergunta?
Que bibliografia devo ver? A bibliografia sobre as views? Devo ver alguma parte da bibliografia sobre views em específico?

Reply:
Livro da Bibliografia do Sigarra: Ullman Jeffrey D.; A First course in database systems. ISBN: 978-0-13-600-637-4